platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -377,6 +377,14 @@ void DisplayServerX11::mouse_set_mode(MouseMode p_mode) {
 
 	// The only modes that show a cursor are VISIBLE and CONFINED
 	bool showCursor = (p_mode == MOUSE_MODE_VISIBLE || p_mode == MOUSE_MODE_CONFINED);
```

Comment:
```suggestion
	bool show_cursor = (p_mode == MOUSE_MODE_VISIBLE || p_mode == MOUSE_MODE_CONFINED);
```

Reply:
From the time before the variable-naming-convention. I have changed it. Thanks!